src/signing/collector/signing_finish_early_strategy.rs

```diff
@@ -14,7 +14,7 @@ pub struct WhenSomeTransactionIsInvalid(pub SignaturesCollectingContinuation);
 
 impl Default for WhenSomeTransactionIsInvalid {
     fn default() -> Self {
-        Self(SignaturesCollectingContinuation::FinishEarly)
+        Self(SignaturesCollectingContinuation::Continue)
```

Comment:
As per discussion https://github.com/radixdlt/sargon-mfa/pull/1#discussion_r1735759277 changed so that we continue collecting signatures even if one transaction has failed.